src/Uno.UWPSyncGenerator/DocGenerator.cs

```diff
@@ -140,6 +140,18 @@ void BuildMemberLists()
 									_sb.AppendParagraph($"In addition, {formattedViewName} has Uno-specific documentation {Hyperlink("here", customDocLink)}.");
 								}
 
+								var galleryLink = GetGalleryLink(viewName);
+								if (galleryLink != null)
+								{
+									_sb.AppendParagraph($"To better understand how {formattedViewName} works, you can use the Gallery {Hyperlink("here", galleryLink)}.");
+								}
+
+								var playgroundLink = GetPlaygroundLink(viewName);
+								if (playgroundLink != null)
+								{
+									_sb.AppendParagraph($"Or run your own tests on {formattedViewName} {Hyperlink("here", playgroundLink)} on {Hyperlink("Playground", playgroundLink)}.");
+								}
```

Comment:
If there's only the playground link, this will look like an odd sentence... :) You'll need to merge both sentences conditional to `galleryLink` and `playgroundLink`.

Reply:
@jeromelaban 

But, the idea is to have links to the Playground and the Gallery.
To add the functionality to the Gallery, I made a PR that allows the link to be redirected to the specific Sample when receiving a hash. https://github.com/unoplatform/Uno.Gallery/pull/1054

And with that, I left two lines here. 
This would emphasize and highlight both features.

Reply:
Yes it is better to unify both links, using something like this:

```suggestion
								var galleryLink = GetGalleryLink(viewName);
								var playgroundLink = GetPlaygroundLink(viewName);
								
								if (galleryLink is not null or playgroundLink is not null)
								{
								    Lists<string> options = new();
								    
								    if(galleryLink is not null)
								    {
								       options .Add($"use the Uno Gallery {Hyperlink("here", galleryLink)}");
								    }			
								    	    
								    if(playgroundLink is not null)
								    {
								       options .Add($"run your own tests on {formattedViewName} {Hyperlink("here", playgroundLink)} on {Hyperlink("Uno Playground", playgroundLink)}");
								    }
								    
								    _sb.AppendParagraph($"To better understand how {formattedViewName} works, you can" + string.Join("or", options) + ".");
								}
```